### DISCUSSION TITLE
packing in .aab

### DISCUSSION THREAD

**Discussion by josquinet** [airsdk/Adobe-Runtime-Support#1702] [General]
Good Morning. I have made an app with animate CC for android. Android no longer allows APKs. You have to pack in .aab. What happens is that I can't find a tutorial on the internet to pack in aab. Something to start with. Could you help me?. Thanks

**Comment by 2jfw** [comment]
Have a look here at the **airsdk.dev website** (credits goto @marchbold):

**Packaging Android App Bundles**
https://airsdk.dev/docs/tutorials/platform/android/packaging-android-app-bundles

@marchbold:
Did you -by the way- already submit "airsdk.dev" to google by [Google Search Console](https://search.google.com/search-console/welcome?action=inspect)?

**Reply by josquinet** [reply]
I haven't sent anything...airsdk.dev is a web page. I don't understand... I'm going to look at your tutorial and if I have doubts about this, I'll ask you. Thanks!

**Reply by 2jfw** [reply]
> I haven't sent anything...airsdk.dev is a web page. I don't understand... I'm going to look at your tutorial and if I have doubts about this, I'll ask you. Thanks!

You misunderstood me - I was asking Michael Archbold from "airsdk.dev" here (thus the @sign:)

**Reply by marchbold** [reply]
@2jfw It's on my list to do :) Will try to get that done this week.

**Reply by marchbold** [reply]
Probably could use an Animate dedicated tutorial there too, will have a look into that.
